Userland/Libraries/LibGL/GLContext.h

```diff
@@ -254,6 +254,10 @@ class GLContext final {
     [[nodiscard]] bool should_append_to_listing() const { return m_current_listing_index.has_value(); }
     [[nodiscard]] bool should_execute_after_appending_to_listing() const { return m_current_listing_index.has_value() && m_current_listing_index->mode == GL_COMPILE_AND_EXECUTE; }
 
+    RefPtr<GPU::Driver> m_driver;
+    NonnullOwnPtr<GPU::Device> m_rasterizer;
```

Comment:
I think this is enough of a hidden detail that it deserves a comment?

Reply:
Absolutely!

Reply:
Thanks!